pkger/clone_resource.go

```diff
@@ -823,7 +823,7 @@ func DashboardToObject(name string, dash influxdb.Dashboard) Object {
 
 	charts := make([]Resource, 0, len(dash.Cells))
 	for _, cell := range dash.Cells {
-		if cell.ID == influxdb.ID(0) {
+		if cell.View == nil {
```

Comment:
curious about this diff. DM me about it

Reply:
ahhh that diff, was a hack I added earlier to prevent pkger from exporting empty cells with nothing in them. Now that I have coupled the cell to the view, I can use its availability to signal if it is there or not